ctf/packages/mobile/src/features/skills-hunt/SkillsHunt.tsx

```
@@ -1,143 +1,510 @@
-import React, { useState } from 'react';
-import { View, Text, FlatList, StyleSheet, TouchableOpacity, TextInput, ScrollView } from 'react-native';
-
-const ROUNDS = [
-  { id: 1, title: 'Web Development Bootcamp', status: 'live', enrolled: 234, capacity: 300, weeks: 8, level: 'Beginner', facilitator: 'Lena H.', color: '#A855F7' },
-  { id: 2, title: 'Trauma-Informed Care Training', status: 'live', enrolled: 189, capacity: 200, weeks: 6, level: 'Intermediate', facilitator: 'Maria G.', color: '#22C55E' },
-  { id: 3, title: 'Financial Literacy Cohort', status: 'upcoming', enrolled: 0, capacity: 150, weeks: 4, level: 'Beginner', facilitator: 'DeShawn W.', color: '#F59E0B' },
-  { id: 4, title: 'Legal Rights Navigator', status: 'upcoming', enrolled: 0, capacity: 100, weeks: 3, level: 'All Levels', facilitator: 'Priya S.', color: '#3B82F6' },
-  { id: 5, title: 'Peer Leadership Program', status: 'completed', enrolled: 120, capacity: 120, weeks: 12, level: 'Advanced', facilitator: 'James T.', color: '#EC4899' },
-];
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
```

ESLint

[error] 'useMemo' is defined but never used.

(no-unused-vars)